src/CashuWallet.ts

```diff
@@ -270,23 +282,30 @@ class CashuWallet {
 			}
 		}
 		const amount = sumProofs(token.proofs) - this.getFeesForProofs(token.proofs);
-		const { payload, blindingData } = this.createSwapPayload(
+		let newOutputData: { send: Array<OutputDataLike> | OutputDataFactory } | undefined = undefined;
+		if (outputData) {
+			newOutputData = { send: outputData };
+		} else if (this._keepFactory) {
+			newOutputData = { send: this._keepFactory };
```

Comment:
why are you using the keepFactory for `send`?

Reply:
Good catch. Unfortunately there is a bug in the current version that requires this: https://github.com/cashubtc/cashu-ts/issues/224